ts/store/reducers/__tests__/debug.test.ts

```diff
@@ -0,0 +1,81 @@
+import { appReducer } from "..";
+import { resetDebugData, setDebugData } from "../../actions/debug";
+import { debugDataSelector } from "../debug";
+
+describe("debug", () => {
+  it("should allow to set only object with string keys", () => {
```

Comment:
This test case does not have any `expect`. Is it intentional? Are you just testing against type errors?

Reply:
Yes, it's just to make sure typing is correct. Do you think it should be removed?

Reply:
I removed them here: https://github.com/pagopa/io-app/pull/6611/commits/07c58a7181eba68728abaef441ebcf9383ddce9d
I don't think we will ever need this tests again, they were useful while developing but now they can be removed.